test/integration/driver-virtualbox.bats

```diff
@@ -316,6 +322,15 @@ findCPUCount() {
   [ "$status" -eq 0  ]
 }
 
+@test "$DRIVER: upgrade should work" {
+  buildMachineWithOldIsoCheckUpgrade
+  [ "$status" -eq 0 ]
```

Comment:
I'm not so great with bats, so I'm open to suggestions of conditionals to more thoroughly test that the upgrade was successfully accomplished if you have ideas.